src/main/webapp/availabilityTable.jsp

```diff
@@ -26,7 +32,7 @@
       <tr>
         <c:forEach items = "${pageScope.list}" var = "day">
           <td onclick="toggleTile(this)" data-utc="${day.get(i).utcEncoding()}" 
-            class="${day.get(i).selected() ? (day.get(i).scheduled() ? 
+              class="${day.get(i).selected() ? (day.get(i).scheduled() ? 
```

Comment:
I know this was how the code originally was, but chained ternary is pretty unreadable. Can this be refactored (maybe calculate this in the java code above?)